declarations/global.d.ts

```diff
@@ -112,3 +112,19 @@ declare const __webpack_public_path__: string;
 type RecursivePartial<T> = {
   [P in keyof T]?: RecursivePartial<T[P]>;
 };
+
+declare namespace JSX {
+  interface IntrinsicElements {
+    div: React.DetailedHTMLProps<
+      React.HTMLAttributes<HTMLDivElement>,
+      HTMLDivElement
+    > & {
+      part?: string;
+    };
+
+    a: React.DetailedHTMLProps<
+      React.AnchorHTMLAttributes<HTMLAnchorElement>,
+      HTMLAnchorElement
+    > & { part?: string };
+  }
+}
```

Comment:
试下这样可以不？不然遇到一个标签就要加一个：

```suggestion
declare namespace React {
  interface HTMLAttributes {
    part?: string;
  }
}
```

Reply:
ok,  可以可以